app/models/labels/plate_label_lbsn_96_lysate.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# Plate label class to print off the 2 labels for the Bioscan LBSN-96 Lysate plate.
+# First label has the standard 96-well plate label information.
+# Second label has the partner plate barcode, plus a reference to the first label.
+# Very specific to this particular pipeline and plate purpose.
+class Labels::PlateLabelLbsn96Lysate < Labels::PlateLabelBase
+  MAX_LENGTH_PARTNER_ID = 8
+  PARTNER_INFO_TEXT = 'PARTNER ID LABEL'
+  PARTNER_ID_SUFFIX = 'SDC'
+
+  # Define the standard first label for the lysate plate
+  def attributes
+    super.merge(barcode: labware.barcode.human)
+  end
+
+  # Define the second label for the lysate plate
+  def intermediate_attributes
```

Comment:
Would it be worth a refactor on the name of these attributes?  It seems like this name was perhaps relevant to another type of label printing but I think in this case you're trying to print two labels to capture more information in the same piece of labware?

I just wouldn't know what intermediate attributes means if I was looking at this code for the first time.

Reply:
Have renamed a couple of those methods in the label class to (I think) make more sense.
I don't really follow why `extra_attributes` is separated (it seems to make an extra 6mm label for 384-well plates but why is that different to additional_attributes?) or what `sprint_attributes` is doing.